drivers/safe/meta.go

```diff
@@ -0,0 +1,47 @@
+package safe
+
+import (
+	"github.com/alist-org/alist/v3/internal/driver"
+	"github.com/alist-org/alist/v3/internal/op"
+)
+
+type Addition struct {
+	// Usually one of two
+	//driver.RootPath
+	//driver.RootID
+	// define other
+
+	FileNameEnc string `json:"filename_encryption" type:"select" required:"true" options:"off,standard,obfuscate" default:"off"`
+	DirNameEnc  string `json:"directory_name_encryption" type:"select" required:"true" options:"false,true" default:"false"`
+	RemotePath  string `json:"remotePath" required:"true" help:"This is where the encrypted data stores"`
+
+	Password        string `json:"password" required:"true" confidential:"true" help:"the main password"`
+	Salt            string `json:"salt" confidential:"true"  help:"If you don't know what is salt, treat it as a second password'. Optional but recommended"`
+	EncryptedSuffix string `json:"encryptedSuffix" required:"true" default:".bin" help:"encrypted files will have this suffix"`
+}
+
+/*// inMemory contains decrypted confidential info and other temp data. will not persist these info anywhere
+type inMemory struct {
+	password string
+	salt     string
+}*/
+
+var config = driver.Config{
+	Name:              "Safe",
```

Comment:
I feel that the driver name is better called `Crypt`, which is the same as rclone

Reply:
lol ok. i just thought safe is an easier word for the crowds. crypt is more international.